stable/datacube-dev-env/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for datacube-wms.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+environment: dev
+image:
+  registry: docker.io
+  repository: opendatacube/k8s-dev
+  tag: "latest"
+  pullPolicy: Always
+
+datacube:
+  configPath: /opt/odc/datacube.conf
+
+database:
+  create: false
+  database: testdb
+  ref_database: ows
+  host: database.local
+  port: 5432
+  adminSecret: dev-eks-datacube
+  existingSecret: dev-eks-datacube
+
+clusterInfo:
+  envName: dev
+
+resources:
+  limits:
+    cpu: "300m"
+    memory: 1024Mi
+
+dockerArgs: [ "/bin/bash", "-c", "tail -f /dev/null" ]
+
+annotations:
+  iam.amazonaws.com/role: dev-eks-datacube-wms
+
+service:
+  type: NodePort
+  port: 80
+
+createdb:
+  dockerArgs: [ "/bin/sh", "-c", "/code/setup/create-db.sh" ]
+  image:
+    registry: docker.io
+    repository: opendatacube/wms
+    tag: latest
+    pullPolicy: Always
+
+setup:
+  dockerArgs:
+    - "/bin/bash"
+    - "-c"
+    - "echo 'CREATE EXTENSION IF NOT EXISTS postgis; \n grant agdc_admin to $ADMIN_USERNAME;' |
+       PGPASSWORD=$ADMIN_PASSWORD psql -h $DB_HOSTNAME -p 5432 -U $ADMIN_USERNAME -d $DB_DATABASE"
+  image:
+    registry: docker.io
+    repository: postgres
+    tag: latest
+    pullPolicy: Always
+
+containerPort: 8080 # Internal port on which pod is running
+
+ingress:
+  ## Enable Ingress.
+  ##
+  enabled: false
+  host:
+
+## Define a TCP liveness probe
+## For a liveness probe, giving up means the pod will be restarted.
+livenessProbe:
+  exec:
+    command:
+      - "ls"
+  initialDelaySeconds: 10
+  periodSeconds: 10
+
+## Define readiness probe
+## For a readiness probe, giving up means not routing traffic to the pod, but the pod is not restarted.
+readinessProbe:
+  exec:
+    command:
+      - "ls"
+  initialDelaySeconds: 5
+  periodSeconds: 10
```

Comment:
These probes have been removed, their values should be too

Reply:
Removed liveness probe and readiness probe from the values.yaml file